_posts/en/newsletters/2021-01-20-newsletter.md

```diff
@@ -0,0 +1,158 @@
+---
+title: 'Bitcoin Optech Newsletter #132'
+permalink: /en/newsletters/2021/01/20/
+name: 2021-01-20-newsletter
+slug: 2021-01-20-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter summarizes posts to the Bitcoin-Dev mailing list
+about payjoin adoption and making hardware wallets compatible with more
+advanced Bitcoin features.  Also included are our regular sections with
+overviews of changes to services and client software, new releases and
+release candidates, and changes to popular Bitcoin infrastructure
+software.
+
+## News
+
+- **Payjoin adoption:** Chris Belcher [posted][belcher payjoin] to the
+  Bitcoin-Dev mailing list a request for people to look for ways to
+  increase [payjoin][topic payjoin] adoption along with a [wiki
+  page][payjoin wiki] tracking the projects that provide either sending
+  or receiving support for payjoin.  One [suggestion][raw payjoin] by
+  Craig Raw was to extend the protocol to allow it to work even when the
+  receiver doesn't operate a server.
+
+- **Making hardware wallets compatible with more advanced Bitcoin features:**
+  Kevin Loaec [started a discussion][loaec hww] on the Bitcoin-Dev
+  mailing list about how hardware wallets could be changed to allow them
+  to handle scripts more complicated than single-sig or multisig.  For
+  example, allowing a hardware wallet to handle in-channel LN payments
+  or payments made from a [vault][topic vaults].  His post does a good
+  job of describing various problems that current hardware wallets can't
+  handle, but he notes that necessary "changes may be very difficult".
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+- **Blockstream announces Jade hardware wallet:**
+  Blockstream's [new Jade hardware wallet][blockstream jade blog] is open source,
+  supports Bitcoin and Liquid networks, and is compatible with Blockstream Green
+  for Android.
+
+- **Coldcard adds payjoin signing:**
+  Coldcard's [3.2.1 release][coldcard 3.2.1] adds [BIP78][] payjoin signing
+  support and various multisig improvements.
+
+- **Mempool v2.0.0 released:**
+  Open source [block explorer][topic block explorers] mempool, which backs the
```

Comment:
Ugh, I wish people would stop naming their companies and projects after concepts that exist in Bitcoin. :(